src/sentry/static/sentry/app/views/settings/incidentRules/ruleConditionsForm.tsx

```diff
@@ -73,6 +74,26 @@ class RuleConditionsForm extends React.PureComponent<Props, State> {
 
   render() {
     const {organization, disabled, onFilterUpdate} = this.props;
+    const {environments} = this.state;
+
+    const environmentList: [string, React.ReactNode][] = defined(environments)
+      ? environments.map((env: Environment) => [env.name, getDisplayName(env)])
+      : [];
+
+    const anyEnvironmentLabel = (
+      <React.Fragment>
+        {t('All Environments')}
+        <div className="all-environment-note">
+          {tct(
+            `This will count events across every environment. For example,
+             having 50 [code1:production] events and 50 [code2:development]
```

Comment:
I think `code1` is the key that the translator would see, no?

Reply:
mm yeah, maybe should call it `productionCode` ? I mean these aren't really so much for the translator right, but idk if we have docs for them to know what they are.

Also we should see if we can update `tct` so you can use the same key for multiple things. I see tct used a lot (maybe just by me?) for adding bold, italics, other stylistic tags. But when I want to add multiple of the same one, I have to do what I did here